react/react1/week1/react-week1/src/App.js

```diff
@@ -0,0 +1,20 @@
+const Todos = [
+  {id: 1, description: "Get out of bed", deadline: "Wed Sep 13 2017" }, 
+  {id: 2, description: "Brush teeth", deadline: "Thu Sep 14 2017"}, 
+  {id: 3, description: "Eat breakfast", deadline: "Fri Sep 15 2017" }
+  // {id: 4, description: "Eat launch", deadline: "Fri Sep 16 2017" }
+];
+
+
+export default function Todo(){
+  return (
+    <div className='Todo'> { 
+      Todos.map((list) => (
+        <div>
+  <h4> {`Id: ${list.id}`}</h4>
```

Comment:
make sure that your code is indented in a consistant manner for better readability of your code. Potentially use a code formatter like prettier which there is an extension for in vs code.